fe/fe-core/src/main/java/com/starrocks/sql/analyzer/InsertAnalyzer.java

```diff
@@ -342,7 +343,19 @@ private static Table getTargetTable(InsertStmt insertStmt, ConnectContext sessio
                 throw unsupportedException(msg);
             }
         }
-
+        if (insertStmt.isIgnore()) {
+            if (!(table instanceof OlapTable)) {
+                throw unsupportedException("Only support insert ignore for olap table");
```

Comment:
Put these error messages into ErrorCode.java.
And print some necessary fields for usability, here we can print table name and table type:
`Only support INSERT IGNORE for olap table, table=$name, type=$table_type`